core/Command/Config/Preset.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OC\Core\Command\Config;
+
+use NCU\Config\Lexicon\Preset as ConfigLexiconPreset;
+use OC\Config\ConfigManager;
+use OCP\IConfig;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Preset extends Command {
```

Comment:
```suggestion
class Preset extends \OC\Core\Command\Base {
```

and then you can use `$this->writeArrayInOutputFormat()` to write the list and get json format for machine readable response for free.